app/controlplane/pkg/data/ent/migrate/migrations/20241210063348.sql

```diff
@@ -0,0 +1,2 @@
+-- Create index "workflowrun_version_id_workflow_id" to table: "workflow_runs"
+CREATE INDEX "workflowrun_version_id_workflow_id" ON "workflow_runs" ("version_id", "workflow_id");
```

Comment:
```suggestion
CREATE INDEX CONCURRENTLY "workflowrun_version_id_workflow_id" ON "workflow_runs" ("version_id", "workflow_id");
```

concurrently allows applying the index without blocking during migration